lib/jobs/run-sku-graph.js

```diff
@@ -76,17 +75,23 @@ function runSkuGraphJobFactory(
                 return;
             }
             if (!node.sku) {
+                // It's okay if there is no SKU, it just means there is
+                // nothing to do.
                 self._done();
                 return;
             }
 
             return waterline.skus.findOne({ id: node.sku })
             .then(function(sku) {
                 if (!sku) {
+                    self._done();
```

Comment:
Shouldn't throw an error? Since sku is assigned to the node so the SKU document should have existed in the database.